tests/python_tests/test_vlm_pipeline.py

```diff
@@ -222,7 +222,10 @@ def test_vlm_with_scheduler_vs_default(config, cache):
 @pytest.mark.nightly
 @pytest.mark.parametrize("model_id", model_ids)
 @pytest.mark.parametrize("system_message", ["", "You are a helpful assistant."])
-def test_vlm_pipeline_chat(model_id, system_message, cache):
+@pytest.mark.parametrize("iteration_images", [[image_links_for_testing[0], image_links_for_testing[0]], [image_links_for_testing[0], image_links_for_testing[2], image_links_for_testing[0]],
+                                              [image_links_for_testing[1], image_links_for_testing[1]], [image_links_for_testing[1], image_links_for_testing[1], image_links_for_testing[1]],
+                                              [image_links_for_testing[2], image_links_for_testing[1]], [image_links_for_testing[2], image_links_for_testing[0], image_links_for_testing[1]]])
```

Comment:
Can you explain the idea behind such iteration_images grouping? One test uses the same image, the other two use different images but in a strange order. It's probably worth adding a code comment instead of answering on GitHub.

Reply:
I have removed several cases (with 1 input image) and add comments, but not sure that it's exhaustive explanation. I wanted to check combinations of inputs, because I faced with fails in such cases, different pictures were used because it's need to check case with multiple pictures as input, but don't want to use 2 picture every time as it's consume time.